### DISCUSSION TITLE
Training a custom Segment Model

### DISCUSSION THREAD

**Discussion by mtbirdsall** [fcakyon/yolov5-pip#160] [Q&A] [answered]
Are there pre-trained weights for a segmentation model, or a tutorial for training the segment models with Yolo?? 

I am assuming the training data & labels would be different between training with bounding boxes and shaded regions--if I am wrong I am willing to read.

Please point me in the right direction. 🤗

**Comment by fcakyon** [comment, marked as answer]
Yes there are pretrained weights for segment models, check this cli command `yolov5 segment train --img 640 --weights yolov5s-seg.pt --epochs 1`

For training it requires polygon annotations to be saved in XML format: https://github.com/ultralytics/yolov5/blob/93b23480f5fb789304bfeb52ba78758b5c80ce2b/segment/tutorial.ipynb

**Reply by mtbirdsall** [reply]
Thank you!!
